include/rocksdb/options.h

```diff
@@ -632,6 +632,25 @@ struct DBOptions {
   // Default: false
   bool track_and_verify_wals_in_manifest = false;
 
+  // EXPERIMENTAL
+  //
+  // If true, various information about predecessor WAL will be recorded in the
```

Comment:
Nit: better to avoid passive voice ("will be recorded in the current WAL" -> "each new WAL will record")

Reply:
Fixed